scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -404,13 +409,13 @@ class FossId internal constructor(
         val existingScan = scans.recentScansForRepository(url, revision).findLatestPendingOrFinishedScan()
 
         val scanCode = if (existingScan == null) {
-            log.info { "No scan found for $url and revision $revision. Creating scan ..." }
+            log.info { "No scan found for $url and revision $revision. Creating scan..." }
 
             val scanCode = namingProvider.createScanCode(projectName)
             val newUrl = if (config.addAuthenticationToUrl) queryAuthenticator(url) else url
             createScan(projectCode, scanCode, newUrl, revision)
 
-            log.info { "Initiating data download ..." }
+            log.info { "Initiating data download..." }
```

Comment:
out of scope, maybe: "Initiating the download..."

Reply:
Done.